projects/burnr/tsconfig.json

```diff
@@ -15,5 +15,6 @@
     "jsx": "react"
   },
   "include": ["src/**/*"],
+  "exclude": ["*node_modules*", "../../node_modules*"],
```

Comment:
why?

Reply:
Cause I thought there is no need to include node modules. I will remove it